bin/test.sh

```diff
@@ -1,17 +0,0 @@
-#!/usr/bin/env bash
-set -e
-
-export DJANGO_SETTINGS_MODULE=settings
-echo 'Beginning Test Run...'
-echo ''
-echo 'Removing *.pyc files'
-find . -name "*.pyc" -exec rm -rf {} \;
```

Comment:
I hope there is no issue with no longer removing pyc files. I would assume not, since tox creates a new directory/environment for each version of Python.